snow/validators/traced_state.go

```diff
@@ -72,3 +72,15 @@ func (s *tracedState) GetValidatorSet(
 
 	return s.s.GetValidatorSet(ctx, height, subnetID)
 }
+
+func (s *tracedState) GetCurrentValidatorSet(
+	ctx context.Context,
+	subnetID ids.ID,
+) (map[ids.ID]*GetCurrentValidatorOutput, uint64, bool, error) {
+	ctx, span := s.tracer.Start(ctx, s.getValidatorSetTag, oteltrace.WithAttributes(
```

Comment:
seems that getSubnetID (Above) is using getValidatorSetTag, but there's a `getSubnetIDTag`.
Seems that we would make a new tag for this method as well

Reply:
Should we fix this in this PR? Seems simple enough to do so.